libraries/AP_NavEKF2/AP_NavEKF2_Control.cpp

```diff
@@ -228,7 +228,7 @@ void NavEKF2_core::setAidingMode()
                    (imuSampleTime_ms - lastTasPassTime_ms > frontend->tiltDriftTimeMax_ms) &&
                    (imuSampleTime_ms - lastRngBcnPassTime_ms > frontend->tiltDriftTimeMax_ms) &&
                    (imuSampleTime_ms - lastPosPassTime_ms > frontend->tiltDriftTimeMax_ms) &&
-                   (imuSampleTime_ms - lastExtNavPassTime_ms > frontend->tiltDriftTimeMax_ms) &&
+                   (imuSampleTime_ms - extNavMeasTime_ms > frontend->tiltDriftTimeMax_ms) &&
```

Comment:
This line can also be removed because both GPS and ExtNav should be updating lastPosPassTime_ms